Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -250,6 +252,9 @@
     LOG_WITH_STREAM(RemoteLayerTree, stream << "RemoteLayerTreeDrawingAreaProxy::commitLayerTree transaction:" << layerTreeTransaction.description());
     LOG_WITH_STREAM(RemoteLayerTree, stream << "RemoteLayerTreeDrawingAreaProxy::commitLayerTree scrolling tree:" << scrollingTreeTransaction.description());
 
+    // Deconstructs at end of function
+    CAFrameLogging CAFrame;
```

Comment:
The variable name is not ideal (style-guide wise) since we want to start with lower letters. Maybe just rename to this?

```suggestion
    CAFrameLogging frameLogging;
```